Mittons.Fixtures/Docker/Attributes/Run.cs

```diff
@@ -10,21 +11,34 @@ public class Run : Attribute, IOptionAttribute
 
         public string Id { get; }
 
+        public bool TeardownOnComplete { get; }
```

Comment:
That's something I've been avoiding because I'm trying to get somebody involved who can review/set standards on documentation before we start getting a mishmash of different people's wording. It's on my radar, though!